src/components/checkbox/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./checkbox.scss";
+
+/**
+ * Checkbox
+ *
+ * Checkbox component
+ *
+ * @return {jsx}
+ */
+export const Checkbox = ({
+  text,
+  isChecked,
+  setIsChecked,
+  disabled,
+  ...rest
+}) => {
+  return (
+    <div className={["checkbox-wrapper", disabled ? "disabled" : ""].join(" ")}>
+      <input
+        type="checkbox"
+        checked={isChecked}
+        onChange={disabled ? () => {} : () => setIsChecked(!isChecked)}
+        className={[isChecked ? "checked" : ""]}
+        {...rest}
+      />
+      {text ? <p className="text">{text}</p> : null}
+    </div>
+  );
+};
+
+Checkbox.propTypes = {
+  /**
+   * Label for the checkbox if needed
+   **/
+  label: PropTypes.string,
```

Comment:
Is 'label' here meant to be "text" ?

Reply:
@sebastian-7DIGIT tbh I think it's good label as most of the times you have label and value where the label is what the user sees and value is what is returned by the checkbox/radio button

Reply:
I agree, 'label' is better. My point is that the 1st prop for the Checkbox component is "text".  I think that should become label.

Reply:
@sebastian-7DIGIT @vasilen-7DIGIT fixed